packages/actor-init-query/package.json

```diff
@@ -47,13 +46,18 @@
     "@comunica/types": "^2.0.1",
     "@rdfjs/types": "*",
     "@types/yargs": "^17.0.2",
+    "arrayify-stream": "^2.0.0",
```

Comment:
The added dependencies here seem incorrect. I think these should be dev dependencies? (or even no deps at all?)

Reply:
Its being triggered by this https://github.com/comunica/comunica/blob/23af30008e7696b81b9f52c267c1f0f6a1c1df9b/packages/actor-init-query/spec/sparql-engine-base.js#L22

The script only excludes subfolders matching `test`, should it exclude `spec` and `__mocks__` as well?

Reply:
Yes, definitely.

But instead of blacklisting, maybe we can use a whitelist? Perhaps we can hook into the `files` entry of the `package.json` file? That should be pretty foolproof.